sdcm/chaos_mesh.py

```diff
@@ -0,0 +1,62 @@
+import json
```

Comment:
also there a directory for k8s related code `sdcm.cluster_k8s`, should be in there, I think...

Reply:
I was looking there, and files there are related more to k8s backends, not test related stuff.
@vponomaryov  what is your opinion?

Reply:
I would place it at `sdcm/utils/k8s.py`.
Almost all the utility code for K8S is there. 
This one is not that big to be somewhere else.

It is definitely not needed in the main `sdcm` catalog.

Reply:
For now it is small, but in the next steps we'll be adding various experiments and their execution code. Things will grow and keeping them in separate module will ease development.

I'll move it to sdcm/utils/chaos-mesh.py

Reply:
I don't mind having it be split into `sdcm/utils/k8s/foo.py`. But, it's place %somewhere% there.

Reply:
I'll move it under sdcm/utils/k8s/